packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {
+  createClientRenderStrictMode,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  fireEvent,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  screen,
+  // @ts-expect-error need to migrate helpers to TypeScript
+  createEvent,
+} from 'test/utils/index';
+import { spy } from 'sinon';
+import { expect } from 'chai';
+import { DataGrid } from '@material-ui/data-grid';
+
+describe('<DataGrid />', () => {
+  // TODO v5: replace with createClientRender
+  const render = createClientRenderStrictMode();
+
+  describe('keyboard', () => {
```

Comment:
Did you consider to flatten the describes too:

```suggestion
describe('<DataGrid /> keyboard', () => {
  // TODO v5: replace with createClientRender
  const render = createClientRenderStrictMode();
```

x the other files.

Reply:
It could make authoring new tests easier with a flatter ree.

Reply:
can do. I don't mind. It depends on how you explore the tests report

Reply:
From what I understand, the generated test report will be almost the same. No strong preference. I was more interested in minimizing the number of tabs.